content/docs/legacy/client-server-api.md

```diff
@@ -0,0 +1,384 @@
++++
+title = "Client Server API"
+aliases = ["/docs/guides/client-server-api"]
++++
+
+## Warning
+
+This guide is badly outdated and should not be used anymore! It is only kept for
+posterity.
```

Comment:
Do we still need this in addition to the warning that's added to all legacy docs?